### DISCUSSION TITLE
[FR] Folder Overview with "Ignore" Settings

### DISCUSSION THREAD

**Discussion by Moyf** [LostPaul/obsidian-folder-overview#10] [Ideas]
Currently, there would be many attachment folders (`assets`) in folder overview:
![image](https://github.com/user-attachments/assets/2d65707c-a61a-493c-ba36-e94b2ae98283)

Could you add an option to "Ignore: xxx" so that we can exclude specific file / folder with regex pattern?

---

BTW, it's strange that even I only choose the Markdown type, it would still show the folders 🤔
![image](https://github.com/user-attachments/assets/df64ec95-eadf-41d0-ad0c-c0170860b0be)

**Comment by LostPaul** [comment]
> BTW, it's strange that even I only choose the Markdown type, it would still show the folders 🤔

It's apparently broken for the file explorer folder overview style but still works with the list type I'll fix it soon.

> Could you add an option to "Ignore: xxx" so that we can exclude specific file / folder with regex pattern?

Currently you can just do this I might add an extra option to the overviews soon.
![C5Jhu0YBY4Ls1L7VTt3X](https://github.com/user-attachments/assets/f2e1bbe4-5619-412c-8716-f94020ae76f0)

**Comment by Moyf** [comment]
> > BTW, it's strange that even I only choose the Markdown type, it would still show the folders 🤔
> 
> It's apparently broken for the file explorer folder overview style but still works with the list type I'll fix it soon.
> 
> > Could you add an option to "Ignore: xxx" so that we can exclude specific file / folder with regex pattern?
> 
> Currently you can just do this I might add an extra option to the overviews soon. ![C5Jhu0YBY4Ls1L7VTt3X](https://private-user-images.githubusercontent.com/70213368/352581296-f2e1bbe4-5619-412c-8716-f94020ae76f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIwNDY4NTcsIm5iZiI6MTcyMjA0NjU1NywicGF0aCI6Ii83MDIxMzM2OC8zNTI1ODEyOTYtZjJlMWJiZTQtNTYxOS00MTJjLTg3MTYtZjk0MDIwYWU3NmYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI3VDAyMTU1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJkNjg2NThmNTgwY2UxY2Q0YmM3ODk4NjRhNmZlMDY5MTBkNDQwM2YwMjQyNmQ0YTdkMmQ4ODIxZGY4MzI5YTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.PLbJulYoecZN2P2OLvG8gOeR2QafLmMXUM2daDnxmEU)

Roger that, thank you!
I really love this plugin and it is essential  for my use of Obsidian. 

Can't wait to see the upcoming updates!  🎉
